platform_versioned_docs/version-24.3/getting-started/studios.mdx

```diff
@@ -0,0 +1,452 @@
+
+#### Create an AWS Batch compute environment 
+
+Studios requires an AWS Batch compute environment. If you do not have an existing compute environment with at least 2 CPUs available, create one with the following attributes:
+
+- **Region**: To reduce costs, your compute environment should be in the same region as your data. To browse the nf-core AWS megatests public data optimally, select **eu-west-1**.
+- **Provisioning model**: Use **On-demand** EC2 instances. 
+- Studios does not support AWS Fargate. Do not enable **Use Fargate for head job**. 
+- At least 2 available CPUs.
+
+#### Add data using Data Explorer
+
+For the purposes of this guide, add the nf-core AWS megatests S3 bucket to your workspace using Data Explorer:
+
+1. From the **Data Explorer** tab, select **Add cloud bucket**. 
+1. Specify the bucket details:
+    - **Provider**: AWS
+    - **Bucket path**: `s3://nf-core-awsmegatests`
+    - A unique **Name** for the bucket, such as `nf-core-awsmegatests`.
+    - **Credentials**: Select **Public** from the dropdown menu.
+    - An optional bucket **Description**.
+1. Select **Add**.
+
+:::info 
+To use your own pipeline data for interactive analysis, add the cloud bucket that contains the results of your nf-core/differentialabundance pipeline run. See [Add a cloud bucket](./quickstart-demo/add-data.mdx#add-a-cloud-bucket) for more information. 
+:::
+
+#### Create an RStudio notebook studio 
+
+From the **Data Studios** tab, select **Add a data studio** and complete the following:
+- In the **Compute & Data** tab:
+    - Select your AWS Batch compute environment. 
+      :::note
+      Data studios compete for computing resources when sharing compute environments. Ensure your compute environment has sufficient resources to run both your pipelines and data studio sessions. The default CPU and memory allocation for a data studio is 2 CPUs and 8192 MB RAM. 
+      :::
+    - Optional: Enter CPU and memory allocations. The default values are 2 CPUs and 8192 MB memory (RAM).
+    - Mount data using Data Explorer: Mount the nf-core AWS megatests S3 bucket, or the directory path that contains the results of your nf-core/differentialabundance pipeline run. 
+- In the **General config** tab:
+    - Select the latest **RStudio** container image template from the list.
+    - Optional: Enter a unique name and description for the data studio. 
+- Select **Add** or choose to **Add and start** the studio immediately.
+- If you chose to **Add** the studio in the preceding step, select **Start** in the options menu, then **Connect** to open the studio in a new browser tab when it is running. 
+
+![Add data studio](./_images/create-ds.gif)
+
+#### Perform the analysis and explore results
+
+1. Configure the RStudio environment with installed packages, including [ShinyNGS](https://github.com/pinin4fjords/shinyngs):
```

Comment:
I have no familiarity with R. Leaving this section to Florian's expert eye for review.